aspnetcore/toc.md

```diff
@@ -125,6 +125,9 @@
 #### [ASP.NET Web API Help Pages using Swagger](tutorials/web-api-help-pages-using-swagger.md)
 #### [Creating backend services for native mobile applications](mobile/native-mobile-backend.md)
 
+### [Create a Razor Pages web app](xref:tutorials/razor-pages/index)
```

Comment:
@tdykstra  it's not empty, it's got the first page. I'll keep adding stages as I complete them. I put it at the bottom because it's not complete yet. When I get 3 or 4 stages done I'll move it to the top. Does that make sense?